lambda/executor.js

```diff
@@ -116,14 +127,17 @@ const runInParallel = async(num, lambdaARN, lambdaAlias, payloads, preARN, postA
         if (invocationResults.FunctionError) {
             throw new Error(`Invocation error (running in parallel): ${invocationResults.Payload} with payload ${JSON.stringify(actualPayload)}`);
         }
+        if(!isNan(sleepBetweenRunsMs) && sleepBetweenRunsMs > 0) {
```

Comment:
Could you please move the `!isNan` check to the `extractDataFromInput` function? that way, we only check it once. Also, you actually need to extract `sleepBetweenRunsMs` from the event object there :)